Cleanups before internal testing (#629)
0xPolygonMiden/miden-base: docs/introduction/get-started/p2p-private.md

    To do this, run:

    ```shell
    miden-client tx new p2id --note-type private <regular-account-id-A> <regular-account-id-B> <faucet-account-id> 50 
    ```

    !!! note

>         The faucet account id is `0xad904b3138d71d3e` and can also be found on the [Miden faucet website](https://ethdenver.polygonmiden.io/) under the title **Miden faucet**.

Reviewer: ```suggestion
        The faucet account id is `0xad904b3138d71d3e` and can also be found on the [Miden faucet website](https://testnet.miden.io/) under the title **Miden faucet**.
```

>     This generates a private Pay-to-ID (`P2ID`) note containing `50` assets, transferred from one account to the other. 

Reviewer: I would leave it `<amount>` because the figures keep changing ...

Author: that's true, but in the example command we use 50. I thought that might be confusing?!

Reviewer: yeah, fair enough :)

    ```

    !!! tip

>         It's possible to use a short version of the note id: 7 characters after the `0x` is sufficient, e.g. `0x6ae613a`

Reviewer: ```suggestion
        It's possible to use a short version of the note id: 7 characters after the `0x` is sufficient, e.g. `0x6ae613a`.
```

    You should now see both accounts containing faucet assets with amounts transferred from `Account A` to `Account B`.

    !!! tip

>         Remember. The original amount was `333` tokens.

Reviewer: i might remove this because the figures keep changing ...

Author: You are right, for now its correct. I have it on the radar

For more information on the Miden client, refer to the [Miden client documentation](https://docs.polygon.technology/miden/miden-client/).

> Wanna do more? [Sending Public Notes](p2p-public.md)

Reviewer: ```suggestion
Wanna do more? [Sending public notes](p2p-public.md)
```
